src/pint/scripts/event_optimize.py

```diff
@@ -519,6 +519,12 @@ def main(argv=None):
         help="Logging level",
         dest="loglevel",
     )
+    parser.add_argument(
+        "--numcores",
+        type=int,
+        default=1.0,
```

Comment:
Python's `multiprocessing` module has a way to query the number of cores, it would be good if it were easy to tell this script "use all my cores, however many there are". Perhaps that should be the default?

Reply:
I was not sure on whether it would be better to let the user specify the number or just run it on all cores by default. I can set it to query and use all cores. If so, should I remove the numcores parser argument?

Reply:
Definitely not. But I would suggest either:

* Default to all cores, but let people specify a smaller number, or
* Default to one core but allow people to specify "all" or "auto" to get all cores.

Reply:
Should be updated now to default to all cores.